<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDiocesis</class>
 <widget class="QWidget" name="dlgDiocesis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti칩n de di칩cesis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filtro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFiltro"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="twDiocesis"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="ckSinComprobar">
         <property name="text">
          <string>Sin comprobar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ckSinUrlHierarchy">
         <property name="text">
          <string>Sin url (hierarchy)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="twObispos"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btModificarDiocesis">
       <property name="text">
        <string>M&amp;odificar di칩cesis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAbrirUrl">
       <property name="text">
        <string>Abrir &amp;url</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbMarcarComprobado">
       <property name="text">
        <string>Co&amp;mprobado</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAnadirDetalles">
       <property name="text">
        <string>A침adir &amp;detalles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbIntroducirObispos">
       <property name="text">
        <string>Introducir obispos</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCerrar">
       <property name="text">
        <string>&amp;Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
